amp/WEB-INF/src/org/digijava/module/aim/util/ActivityVersionUtil.java

```diff
@@ -11,7 +12,7 @@
 import java.util.HashSet;
 import java.util.Iterator;
 import java.util.Set;
-
+//import org.digijava.kernel.request.TLSUtils;
```

Comment:
if an import is not in used or duplicate in order to keep code clean we should remove it instead of commenting. we can trace changes using GitHub history